### ISSUE TITLE
Slot 3 ROM space does not float when Uthernet is enabled

### ISSUE THREAD

**Issue by txgx42** [AppleWin/AppleWin#418]
The slot 3 C3xx ROM space should access the floating bus when the IIe 80 column firmware is switched out. It presently being filled with $00 when Uthernet is enabled. It works correctly if Uthernet is disabled. I also verified that the bus does float on a real IIe with Uthernet card in slot 3.

This causes an issue under ProDOS 1.1 specifically. This version checks for presence of a card in slot 3 by switching out the 80-col firmware and testing for the floating bus. If the value it reads does not change, it will assume a card is present and leave the internal firmware switched out. In Applewin, booting into ProDOS 1.1 and then doing PR#3 will hang if Uthernet is enabled. 

Simple test is to use the monitor to switch out the firmware by writing to C00B and then examine C3xx.

![image](https://cloud.githubusercontent.com/assets/19871324/25976734/d218f508-3684-11e7-8ef5-3ce8121d40d7.png)

**Comment by txgx42** [comment]
Sure, I am currently using 1.26.2.3.

**Comment by tomcw** [comment]
Screenshot after applying fix:

![tfe-floating-bus](https://cloud.githubusercontent.com/assets/6696896/26029378/68dc6bbc-382b-11e7-93ac-240154238c6d.png)

**Comment by tomcw** [comment]
Re-opening, as a2audit v1.04 is failing:
```
687B:
  LDA #$1A
  LDA $C300
  STA $C00B ; ENABLE SLOTC3ROM
```

**Comment by tomcw** [comment]
Tracking this a2audit v1.04 failure in #423 (so closing this issue again).

**Comment by tomcw** [comment]
I've pre-released a new AppleWin 1.26.2.4 version [here](https://github.com/AppleWin/AppleWin/releases/tag/v1.26.2.4) that fixes this.
